src/profile-logic/process-profile.js

```diff
@@ -1882,6 +1886,8 @@ export async function unserializeProfileOfArbitraryFormat(
       const arrayBuffer: ArrayBuffer = (arbitraryFormat: any);
       if (isArtTraceFormat(arrayBuffer)) {
         arbitraryFormat = convertArtTraceProfile(arrayBuffer);
+      } else if (isSimpleperfTraceFormat(arrayBuffer)) {
+        arbitraryFormat = convertSimpleperfTraceProfile(arrayBuffer);
```

Comment:
it would be good to use dynamic imports so that `import/simpleperf` would be lazyloaded. This way we wouldn't have protobufjs and long.js in the main bundle (they're pretty big!).